lua/telescope/entry_manager.lua

```diff
@@ -130,14 +130,16 @@ function EntryManager:add_entry(picker, score, entry)
     return
   end
 
+  -- may need the prompt for tiebreak
+  local prompt = picker:_get_prompt()
```

Comment:
`_get_prompt` is not cheep. because `add_entry` will be called a lot of times. Will pass in the prompt as argument